features/support/env.rb

```diff
@@ -150,5 +150,15 @@ def extract_process_id(data)
   end
 
   expect(binaries.size).to eq(1)
-  expect(File.executable?(binaries[0])).to eq(true)
+  expect(File.executable?(binaries.first)).to eq(true)
+end
+
+Then(/^the binary "([^"]*)" should be installed with version "([^"]*)"$/) do |binary, version|
+  binaries = []
+  Find.find("#{ENV['VAGRANT_HOME']}/data/service-manager/bin") do |path|
```

Comment:
> I don't know but #{ENV['VAGRANT_HOME']}/data/service-manager/custom-path" is not working in command

What's the error? My guess is that the variable substitution does not work. Did you debug it? Check _aruba/cucumber/command.rb_ and _aruba/api/command.rb_. If backtick does not work, we can use _system_ and write our own step definition. Have a go at it.